winui/Kanban/Getting-started.md

```diff
@@ -0,0 +1,241 @@
+            string path = @"ms-appx:///";
+
+            KanbanModel taskDetail = new KanbanModel();
+            taskDetail.Title = "UWP Issue";
+            taskDetail.Id = "651";
+            taskDetail.Description = "Crosshair label template not visible in UWP";
+            taskDetail.Category = "Open";
+            taskDetail.IndicatorColorKey = "High";
+            taskDetail.Tags = new List<string>() { "Bug Fixing" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri("ms-appx:///Assets/Kanban/People_Circle1.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "WinUI Issue";
+            taskDetail.Id = "646";
+            taskDetail.Description = "AxisLabel cropped when rotating the axis label";
+            taskDetail.Category = "Open";
+            taskDetail.IndicatorColorKey = "Low";
+            taskDetail.Tags = new List<string>() { "Bug Fixing" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle2.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "Kanban Feature";
+            taskDetail.Id = "25678";
+            taskDetail.Description = "Provide drag and drop support";
+            taskDetail.Category = "In Progress";
+            taskDetail.IndicatorColorKey = "Low";
+            taskDetail.Tags = new List<string>() { "New control" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle3.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "New Feature";
+            taskDetail.Id = "29574";
+            taskDetail.Description = "Dragging events support for Kanban";
+            taskDetail.Category = "Closed";
+            taskDetail.IndicatorColorKey = "Normal";
+            taskDetail.Tags = new List<string>() { "New Control" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle4.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+            return taskDetails;
+        }
+
+        #endregion
+    }
+
+{% endhighlight %}
+
+### Bind Data to the Kanban Control
+
+To bind a data source to the `SfKanban` control, set the `ItemsSource` property as demonstrated in the example below. This property connects the collection of `KanbanModel` objects to the Kanban board, enabling the display of tasks.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<Grid>
+    <kanban:SfKanban x:Name="sfKanban"
+                     ItemsSource="{Binding TaskDetails}"/>
+    <Grid.DataContext>
```

Comment:
Move DataContext code to under grid initialization

Reply:
Moved DataContext code under grid initialization